client/landing/stepper/declarative-flow/internals/steps-repository/domain-search/index.tsx

```diff
@@ -20,9 +20,17 @@ import FormattedHeader from 'calypso/components/formatted-header';
 import { SIGNUP_DOMAIN_ORIGIN } from 'calypso/lib/analytics/signup';
 import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
 import { getSuggestionsVendor } from 'calypso/lib/domains/suggestions';
-import { domainManagementTransferToOtherSite } from 'calypso/my-sites/domains/paths';
+import {
+	domainAddNew,
+	domainManagementList,
+	domainManagementRoot,
+	domainManagementTransferIn,
+	domainManagementTransferToOtherSite,
+	domainMapping,
+} from 'calypso/my-sites/domains/paths';
```

Comment:
@hambai this step is used within `/setup/domain`, which is used by the Hosting Dashboard.

Could you please give me a list of URLs so I can redirect to the proper places from the Hosting Dashboard? Thanks.